<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyCommittingTool</class>
 <widget class="QWidget" name="KeyCommittingTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="input_text_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cipher_type_combo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>Custom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AEGIS-128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AEGIS-128L</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AEGIS-256</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rocca</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rocca-S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tiaoxin-346</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="analyze_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="result_text_edit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>441</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="equation_text_edit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>441</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AEAD Scheme</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Block Types (ex : 3)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Blocks (ex : 3 4 6)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_text_edit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Block Names (ex : U V W)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_text_edit_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of AD Types (ex : 3)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_text_edit_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of ADs (ex : 1 1 1)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_text_edit_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AD Names (ex : a b c)</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_text_edit_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Round Functions (ex : U0+a0+A(U2))</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_text_edit_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
